.github/workflows/unversioned_pages.yml

```
@@ -0,0 +1,33 @@
+name: Copy unversioned pages
+
+on:
+  push:
+    branches:
+      - gh-pages
```

Since I last looked at this PR I learned about [path filters](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) from some of our main repo workflows like the [citation workflow](https://github.com/napari/napari/blob/a9c25e63b0092bc4f941d8bf25dae599a39c71e1/.github/workflows/citation_cff_validate.yml#L7). So I would suggest adding:

```suggestion
      - gh-pages
    paths:
      - 'dev/developers/**'
      - 'dev/index.html'
```

This should save some CI time!